qemu/tests/cfg/hpt.cfg

```diff
@@ -0,0 +1,18 @@
+# Notes:
+#     These test cases are only for HPT guest,which include Power8 guest
+#     and Power8 guest with compatible mode running on Power9 host.
+- hpt:
+    type = hpt_basic
+    no Host_RHEL.6
+    no RHEL.5 RHEL.6
+    only pseries
+    getcommand ="cat /sys/kernel/debug/powerpc/hpt_order"
+    setcommand ="echo %s > /sys/kernel/debug/powerpc/hpt_order"
+    error_check = "No such file or directory"
```

Comment:
For line 9 ~ 11, I think should be removed then executed in case.

Reply:
Remove those parameters and updated them to code directly